mr/src/main/java/org/apache/iceberg/mr/Catalogs.java

```diff
@@ -68,10 +71,11 @@ private static Table loadTable(Configuration conf, String tableIdentifier, Strin
     if (catalog.isPresent()) {
       Preconditions.checkArgument(tableIdentifier != null, "Table identifier not set");
       return catalog.get().loadTable(TableIdentifier.parse(tableIdentifier));
+    } else {
+      Preconditions.checkArgument(tableLocation != null, "Table location not set");
+      return new HadoopTables(conf).load(tableLocation);
```

Comment:
Looks like this change is not needed. Could you roll it back?

Reply:
I missed this one. Sorry :(